documentation/repository-configuration/index.md

```diff
@@ -211,10 +281,10 @@ the current connection (thread-safely).
 <h3>Schema</h3>
 JaVers creates four tables in SQL database:
 
-*  `jv_global_id` — domain object identifiers,
-*  `jv_commit` — JaVers commits metadata,
-*  `jv_commit_property` — commit properties,
-*  `jv_snapshot` — domain object snapshots.
+- `jv_global_id` — domain object identifiers,
+- `jv_commit` — JaVers commits metadata,
+- `jv_commit_property` — commit properties,
+- `jv_snapshot` — domain object snapshots.
```

Comment:
looks like you did a sort of auto-formatting here, could you please revert it and leave only the new content?

Reply:
The incorrect auto-formatting has been reverted.